Fuse.pretty/Fuseholder_Clip-5x20mm_Keystone_3512P_Inline_P23.62x7.27mm_D1.02x2.41x1.02x1.57mm_Horizontal.kicad_mod

```diff
@@ -0,0 +1,50 @@
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 6.76 -2.5) (end 16.86 -2.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 16.86 2.5) (end 6.76 2.5) (layer F.Fab) (width 0.1))
+  (fp_line (start 16.86 -3.175) (end 16.86 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 16.86 3.175) (end 23.11 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 23.11 3.175) (end 23.11 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 23.11 -3.175) (end 16.86 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.51 -3.175) (end 6.76 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 6.76 -3.175) (end 6.76 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 6.76 3.175) (end 0.51 3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 0.51 3.175) (end 0.51 -3.175) (layer F.Fab) (width 0.1))
+  (fp_line (start 16.86 -3.175) (end 16.86 -1.4) (layer F.SilkS) (width 0.12))
+  (fp_line (start 16.86 3.175) (end 23.11 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 23.11 -3.175) (end 16.86 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 16.86 2.5) (end 6.76 2.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start 16.86 -2.5) (end 6.76 -2.5) (layer F.SilkS) (width 0.12))
+  (fp_line (start 23.11 1.8) (end 23.11 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.76 -3.175) (end 0.51 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 0.51 -3.175) (end 0.51 -1.8) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.76 3.175) (end 0.51 3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.76 -3.175) (end 6.76 -1.4) (layer F.SilkS) (width 0.12))
+  (fp_line (start -1.35 -3.43) (end 24.97 -3.43) (layer F.CrtYd) (width 0.05))
+  (fp_line (start -1.35 -3.43) (end -1.35 3.43) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 24.97 3.43) (end 24.97 -3.43) (layer F.CrtYd) (width 0.05))
+  (fp_line (start 24.97 3.43) (end -1.35 3.43) (layer F.CrtYd) (width 0.05))
+  (fp_text user %R (at 11.81 0) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_line (start 0.51 3.175) (end 0.51 1.8) (layer F.SilkS) (width 0.12))
+  (fp_line (start 23.11 -1.8) (end 23.11 -3.175) (layer F.SilkS) (width 0.12))
+  (fp_line (start 6.76 3.175) (end 6.76 1.4) (layer F.SilkS) (width 0.12))
+  (fp_line (start 16.86 3.175) (end 16.86 1.4) (layer F.SilkS) (width 0.12))
+  (pad 2 thru_hole oval (at 16.35 0) (size 2 2.6) (drill oval 1.02 1.57) (layers *.Cu *.Mask))
+  (pad 2 thru_hole oval (at 23.62 0) (size 2 3.4) (drill oval 1.02 2.41) (layers *.Cu *.Mask))
+  (pad 1 thru_hole oval (at 7.27 0) (size 2 2.6) (drill oval 1.02 1.57) (layers *.Cu *.Mask))
+  (pad 1 thru_hole oval (at 0 0) (size 2 3.4) (drill oval 1.02 2.41) (layers *.Cu *.Mask))
+  (model ${KISYS3DMOD}/Fuse.3dshapes/Fuseholder_Clip-5x20mm_Keystone_3512P_Inline_P23.62x7.27mm_D1.02x2.41x1.02x1.57mm_Horizontal.wrl
+    (at (xyz 0 0 0))
+    (scale (xyz 1 1 1))
+    (rotate (xyz 0 0 0))
+  )
+)
```

Comment:
Courtyard clearance is 0.35mm from pads and 0.26mm from fab

Reply:
Fixed to 0.25mm on L and R sides from the pad. Top and bottom looked good. Did I miss anything?